### PR TITLE
OBSDOCS-695:Update Network Observability doc links and topic map posi…

### PR DESCRIPTION
**Pull request by skrthomas**
Version(s):

4.12+
Issue:

https://issues.redhat.com/browse/OBSDOCS-695
Link to docs preview:

https://73554--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/network_observability/network-observability-operator-release-notes
QE review:
- QE review not required. This change is part of a broad effort to reorganize Observability documentation under an Observability category. This change has been approved by different levels of PM. See related PR, https://github.com/openshift/openshift-docs/pull/71248 and
image

Additional information:

This PR changes the topic map file, turns the assembly directory for `network_observability` into a subdirectory of `observability`, and updates the affected xrefs.

### CODE REVIEW

**Review by mramendi — APPROVED** [review]
All the changes look good to me for peer review and I have checked links out of the network observability text and they work fine.

However: I think there might be more work to do. I have run a full-text search on main and there are files that have xrefs into network_observability and that you don't update. Will these links end up broken?

The files I found are:

networking/networking-dashboards.adoc
networking/networking-operators-overview.adoc
observability/index.adoc (this one even has a comment that this update needs to be made)

**Comment by mramendi** [comment]
As I am in peer review induction, @JoeAldinger will finish this review

**Review by JoeAldinger — COMMENTED** [review]
I _think_ xrefs to the assembly files are going to need to be updated too. Sorry I couldn't be of more help.

**Comment by JoeAldinger** [comment]
cc @kalexand-rh not sure what is going on here and why the build is passing.

**Comment by skrthomas** [comment]
Thanks @mramendi. I changed the links in networking/networking-dashboards.adoc and networking/networking-operators-overview.adoc to have a link path: xref:../observability/network_observability. However, in all of my other assembly files, I tried to do xref:../observability/network_observability or ../../observability/network_observability, but I kept getting build errors. When I talked to Joe, he suggested doing ../../network_observability. It didn't break the PR he pulled down, so hopefully I'm going to try that on this PR and see.

File: observability/network_observability/network-observability-operator-release-notes.adoc [hunk 1]
```diff
@@ -199,7 +199,7 @@ Network Observability Operator can now run on `s390x` architecture. Previously i
 === Bug fixes
 * Previously, the Prometheus metrics exported by Network Observability were computed out of potentially duplicated network flows. In the related dashboards, from *Observe* -> *Dashboards*, this could result in potentially doubled rates. Note that dashboards from the *Network Traffic* view were not affected. Now, network flows are filtered to eliminate duplicates prior to metrics calculation, which results in correct traffic rates displayed in the dashboards. (link:https://issues.redhat.com/browse/NETOBSERV-1131[*NETOBSERV-1131*])
```
[the remark this answers is not in the document]
Comment (skrthomas) [reply observability/network_observability/network-observability-operator-release-notes.adoc]: Addressed

**Pull request merged**